stdlib/_interpqueues.pyi

```diff
@@ -0,0 +1,16 @@
+from typing import SupportsIndex
+
+class QueueError(RuntimeError): ...
+class QueueNotFoundError(QueueError): ...
```

Comment:
There also `QueueEmpty` and `QueueFull`:

https://github.com/python/cpython/blob/722229e5dc1e499664966e50bb98065670033300/Modules/_interpqueuesmodule.c#L158-L159

Reply:
I'm not seeing these actually get exported, might be because of:

```c
// QueueEmpty and QueueFull are set by set_external_exc_types().
state->QueueEmpty = NULL;
state->QueueFull = NUL
```

3.13.0b2

```python
>>> import _interpqueues as iq
>>> iq.QueueEmpty
Traceback (most recent call last):
  File "<python-input-5>", line 1, in <module>
    iq.QueueEmpty
AttributeError: module '_interpqueues' has no attribute 'QueueEmpty'
>>> iq.QueueFull
Traceback (most recent call last):
  File "<python-input-6>", line 1, in <module>
    iq.QueueFull
AttributeError: module '_interpqueues' has no attribute 'QueueFull'
>>>
```